src/dimacsparser.h

```diff
@@ -498,10 +538,11 @@ bool DimacsParser<C>::parse_DIMACS(T input_stream, const bool _strict_header)
     if ( !parse_DIMACS_main(in)) {
         return false;
     }
-
+    
```

Comment:
Please use an editor that automatically removes spaces from the end of lines. This change is just noise. Please fix. Thanks!